tests/Makefile

```diff
@@ -71,6 +71,9 @@ all: $(PROGS) $(UTILS) $(PLAT_UTILS) $(FUNCS) $(SPECIAL_FUNCS) $(TEST_TARGET)
 
 tests_only: $(PROGS) $(TEST_TARGET)
 
+GITIGN = .gitignore
+MKGITIGN = [ -f "$(GITIGN)" ] || echo "$(GITIGN)" > $(GITIGN); grep -q '^\$@$$' $(GITIGN) || echo "$@" >> $(GITIGN)
```

Comment:
All it does is autogenerate a .gitignore file automatically updated with all built executabled.

.gitignore may be version controlled, however, this is optional. There is a reason why the user must do "git add" on it before it's version controlled.